app/services/hyrax/adapters/nesting_index_adapter.rb

```diff
@@ -65,14 +68,16 @@ def self.each_perservation_document_id_and_parent_ids(&block)
       # @param pathnames [Array<String>]
       # @return Hash - the attributes written to the indexing layer
       def self.write_document_attributes_to_index_layer(id:, parent_ids:, ancestors:, pathnames:)
-        solr_doc = ActiveFedora::Base.find(id).to_solr # What is the current state of the solr document
-
-        # Now add the details from the nesting indexor to the document
-        solr_doc[solr_field_name_for_storing_ancestors] = ancestors
-        solr_doc[solr_field_name_for_storing_parent_ids] = parent_ids
-        solr_doc[solr_field_name_for_storing_pathnames] = pathnames
-        ActiveFedora::SolrService.add(solr_doc, commit: true)
-        solr_doc
+        ActiveFedora::Base.uncached do
```

Comment:
I think having uncached around the whole block might be confusing.  It's only affecting line 72, right?  Can we minimize the scope?